mod_ci/models.py

```diff
@@ -15,6 +15,17 @@
 from mod_test.models import Test, TestPlatform
 
 
+class BlockedUsers(Base):
+    __tablename__ = 'BlockedUsers'
+    # userID refers to the ID from https://api.github.com/users/your_username
+    userID = Column(Integer, primary_key=True)
+    comment = Column(String)
+
+    def __repr__(self):
+        return "<BlockedUsers(userID='{TheirID}', comment='{Info}')>".format(
```

Comment:
please use lowercase for the named arguments.

Reply:
Sorry, a moment please.